pkgs/tools/package-management/lix/default.nix

```diff
@@ -137,7 +153,39 @@ lib.makeExtensible (self: {
     };
   };
 
-  latest = self.lix_2_91;
+  lix_2_92 = self.makeLixScope {
+    lix-args = rec {
+      version = "2.92.0";
+
+      src = fetchFromGitHub {
+        owner = "lix-project";
+        repo = "lix";
+        rev = version;
+        hash = "sha256-CCKIAE84dzkrnlxJCKFyffAxP3yfsOAbdvydUGqq24g=";
+      };
+
+      cargoDeps = rustPlatform.fetchCargoVendor {
+        name = "lix-${version}";
+        inherit src;
+        allowGitDependencies = false;
+        hash = "sha256-YMyNOXdlx0I30SkcmdW/6DU0BYc3ZOa2FMJSKMkr7I8=";
+      };
+    };
+
+    nix-eval-jobs-args = rec {
+      version = "2.92.0";
+      src = fetchgit {
+        url = "https://git.lix.systems/lix-project/nix-eval-jobs.git";
+        rev = version;
+        hash = "sha256-tPr61X9v/OMVt7VXOs1RRStciwN8gDGxEKx+h0/Fg48=";
+      };
```

Comment:
This could use fetchFromGitea which is compatible with Forgejo. See https://github.com/NixOS/nixpkgs/blob/250b695f41e0e2f5afbf15c6b12480de1fe0001b/pkgs/applications/audio/listenbrainz-mpd/default.nix#L21-L27 as an example.